cmd/ccienv/main.go

```diff
@@ -35,8 +35,11 @@ func handleErr(err error) {
 	}
 }
 
-func extractRepoName(repo string) (string, string, error) {
-	r := regexp.MustCompile(`.+github\.com[:/]([^/]+)/(.+)\.git/?$`)
+func extractRepoName(uri string) (string, string, error) {
+	repoURI := strings.TrimSpace(string(uri))
+	rmSuffix := regexp.MustCompile(`.git$`)
```

Comment:
```suggestion
	rmSuffix := regexp.MustCompile(`.git/?$`)
```
(全部に対応するつもりはないけど)trailing slashも一般的のようなので対応しておきたい。
テストケース2番目で対応しているつもりだったけどできていなかった...

Reply:
ミスっぽい？ので足しました
https://github.com/threepipes/circleci-env/pull/14/commits/ea816ee48a65c7920d6ce3ee778d588e00a17833